### PR TITLE
fix(core/group): use correct cursors

### PR DESCRIPTION
**Pull request by ridvandmrc**
…d by user

## 💡 What is the current behavior?

- group-item cursor is default which is not correct

GitHub Issue Number: #1207 

## 🆕 What is the new behavior?

- initially, group item cursor is pointer
- and also, the user can override it because it inherits from host element

## 🏁 Checklist

A pull request can only be merged if all of these conditions are met (where applicable):

- [X] 🦮 Accessibility (a11y) features were implemented
- [X] 🗺️ Internationalization (i18n) - no hard coded strings
- [X] 📲 Responsiveness - components handle viewport changes and content overflow gracefully
- [X] 📄 Documentation was reviewed/updated (`pnpm run docs`)
- [X] 🧪 Unit tests were added/updated and pass (`pnpm test`)
- [X] 📸 Visual regression tests were added/updated and pass ([Guide](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md#visual-regression-testing))
- [X] 🧐 Static code analysis passes (`pnpm lint`)
- [X] 🏗️ Successful compilation (`pnpm build`, changes pushed)

### CODE REVIEW

File: packages/core/src/components/group-item/group-item.scss [hunk 1]
```diff
@@ -20,6 +20,7 @@ $focusBorderWidth: 1px;
   display: flex;
   min-height: 2.25rem;
   height: 2.25rem;
+  cursor: pointer;
```
Comment (nuke-ellington) [inline packages/core/src/components/group-item/group-item.scss:23]: Setting the cursor on the button should suffice

File: packages/core/src/components/group-item/group-item.scss [hunk 2]
```diff
@@ -32,6 +33,7 @@ $focusBorderWidth: 1px;
     background-color: var(--theme-group-item--background);
     border: 1px solid transparent;
     color: var(--theme-color-std-text);
+    cursor: inherit;
```
Comment (nuke-ellington) [inline packages/core/src/components/group-item/group-item.scss:36]: Only show _pointer_ if _suppressSelection_ is not set!

**Review by nuke-ellington — CHANGES_REQUESTED** [review]

File: packages/core/src/components/group-item/group-item.scss (hunk 1 above)
Comment (ridvandmrc) [reply packages/core/src/components/group-item/group-item.scss:23]: Yes, but in this way , we give an option to manipulate to change cursor.
and also ionic is following same approach:
https://github.com/ionic-team/ionic-framework/tree/main/core/src/components

Comment (nuke-ellington) [reply packages/core/src/components/group-item/group-item.scss:23]: I get your argument, but we will develop a guide line how to handle cursors in the whole design system. Until that will don't want to change the this.

File: packages/core/src/components/group-item/group-item.scss (hunk 2 above)
Comment (ridvandmrc) [reply packages/core/src/components/group-item/group-item.scss:36]: yes I will do it

Comment (nuke-ellington) [reply packages/core/src/components/group-item/group-item.scss:36]: Great, can you do it today? :D
We are planning to release this week.

Comment (ridvandmrc) [reply packages/core/src/components/group-item/group-item.scss:36]: of course I will do it today :)

**Review by nuke-ellington — DISMISSED** [review]

**Review by nuke-ellington — APPROVED** [review]

**Pull request merged**